HPXMLtoOpenStudio/measure.rb

```diff
@@ -153,6 +153,7 @@ def run(model, runner, user_arguments)
         if hpxml.buildings.size > 1
           # Create the model for this single unit
           unit_model = OpenStudio::Model::Model.new
+          schedules_file.set_unit_model(unit_model) # otherwise we find Schedule:File in model without External:File
```

Comment:
This allows, e.g., `base-two-buildings-detailed-schedules.xml`, to run successfully.